plugins/generator-1.18.2/forge-1.18.2/templates/block/block.java.ftl

```diff
@@ -678,7 +684,13 @@ public class ${name}Block extends
 				<#if data.tintType == "Grass">
 					return GrassColor.get(0.5D, 1.0D);
 				<#elseif data.tintType == "Foliage">
-					return FoliageColor.getDefaultColor();
+					return FoliageColors.getDefault();

```

Comment:
In 1.18 this method is called `getDefaultColor()`

Reply:
it seems 1.16.5 alos uses the Colour() part at the end I seemed to of looked at the wrong reff when it come to the call so I will also change that call to Color on them all.

I will also test 1.18.2 little more thanks.

Reply:
Yep seems https://nekoyue.github.io/ForgeJavaDocs-NG/javadoc/1.16.5/net/minecraft/world/FoliageColors.html is wrong on the changes to 1.16.5